karma.conf.js

```diff
@@ -24,7 +24,7 @@ module.exports = config => {
       // use different import syntaxes so it's not easy to just require it here.
       // Maybe this could be put into a JSON file, but the include in the module
       // rules is dynamic.
-      entry: ["babel-polyfill", "./src/index.js"],
+      entry: ["@babel/polyfill", "./src/index.js"],
```

Comment:
Question:
+ Do we still need the polyfill here? Looking at the babel docs it looks like we can specify `entry` on the `useBuiltIns` option for `preset-env` ?  https://babeljs.io/docs/en/babel-polyfill/#usage-in-node--browserify--webpack

I don't want to send us on a wild goose chase so if this is working, then Im totes happy to keep it! I got lost in trying to fix this and it seemed like a lot of heartache for not a big difference in code change.

Reply:
Good idea.  Definitely seems like that should work!